static/docs/get-started/add-files.md

```diff
@@ -2,24 +2,22 @@
 
 DVC allows storing and versioning data files, ML models, directories,
 intermediate results with Git, without checking the file contents into Git.
-Let's get a sample dataset to play with:
+Let's get a dataset example to play with:
 
 ```dvc
 $ mkdir data
-$ wget https://data.dvc.org/get-started/data.xml -O data/data.xml
+$ cd data
+$ dvc get https://github.com/iterative/dataset-registry \
```

Comment:
maybe just name it iterative/datasets?

Reply:
Maybe. Opened https://github.com/iterative/dataset-registry/issues/1